ouroboros-network/src/Ouroboros/Network/Socket.hs

```diff
@@ -249,6 +253,7 @@ beginConnection muxTracer handshakeTracer encodeData decodeData acceptVersion fn
         let muxTracer' = Mx.WithMuxBearer (show peerid) `contramap` muxTracer
```

Comment:
it's outside of this PR, but it would be good to remove the `show peerid` and just store `peerid`. Do we have an issue for that?

Reply:
I don't think there is an issue for it, but I remember the comment.

Reply:
Could you add one so we don't forget about it.

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/1120 found it.